act-framework.bs

```diff
@@ -42,10 +42,11 @@ Rule Description {#structure-description}
 
 Each ACT Rule MUST have a description that:
 
-* Is in plain language;
-* Explains the accessibility requirement being tested;
-* Lists any limitations and assumptions for the test;
-* Explains the failure conditions and their reasons.
+* Is in plain language and provides a brief description of what the rule does.
+* Explains the accessibility requirement being tested such as the WCAG SC or the technique the rule maps to; For example WCAG 2.0 Technique H67.
+* Lists any limitations and assumptions for the test, the test environment, technologies being used or the subject being tested; For example: A rule for 1.4.1: Use of Color has to make an assumption that CSS properties are used to make a link visually evident. Typical examples are: background, border, color, font, or text decoration.
+* Provides the scope or the context of the rule; 
+* Explains the failure conditions and their reasons. For example the rule triggers on all images on the page that has no alt attribute defined.
```

Comment:
> the rule triggers o all the images on the page that has no alt attribute defined

do it mean the rule is applied to all such images, or returns a `fail` to all such images? (in other words, is it an example of the rule scope and context, or of the rule failure condition?)